nimbus_generator/example/lib/model/response/login_response_model.dart

```diff
@@ -0,0 +1,44 @@
+import 'package:nimbus4flutter/nimbus4flutter.dart';
+import 'package:nimbus_annotation/nimbus_annotation.dart';
+
+part 'login_response_model.g.dart';
+
+/// "schema": {
+//      "header": {
+//          "Common": [
+//              "result_code",
+//              "result_message"
+//          ],
+//          "Login": [
+//              "user_id",
+//              "session_id",
+//          ]
+//      }
+//  },
+@DatasetSerializable(name: 'ResponseDataSet')
+class LoginResponseModel {
+  LoginResponseModel({this.login});
+
+  factory LoginResponseModel.fromJson(Map<String, dynamic> json) =>
+      _$LoginResponseModelFromJson(json);
+
+  Map<String, dynamic> toJson() => _$LoginResponseModelToJson(this);
+
+  @DatasetHeader()
+  final LoginResponseRecord? login;
+}
+
+@RecordSerializable()
+class LoginResponseRecord {
+  LoginResponseRecord({this.userId, this.sessionId});
+
+  factory LoginResponseRecord.fromJson(Map<String, dynamic> json) =>
+      _$LoginResponseRecordFromJson(json);
+
+  static RecordSchema? get schema => _$LoginResponseRecordSchema();
+
+  Map<String, dynamic> toJson() => _$LoginResponseRecordToJson(this);
+
+  final String? userId;
+  final String? sessionId;
+}
```

Comment:
Record is an extension of Map, and RecordList is an extension of List<Record>, but looking at the output of this generator, it seems that it is converted to a bean that contains a Record. Am I right?

Reply:
Yes, I am doing such a thing.

I think it must be the schema, I will update them

Reply:
I renamed Record to Schema

Reply:
One of the goals of this library is to achieve dynamic DTOs by defining a schema in the first place, without having to develop concrete classes to define input and output DTOs.

So, defining a static DTO class and then generating a class that encompasses the dynamic DTOs of this library by adding decorations to it is in itself an act that hinders the purpose of the library.

Therefore, I would like to hear the intent of this.